File: qqbot-app/src/main/java/com/mieai/qqbot/app/config/AppSecretEncryptionProperties.java
package com.mieai.qqbot.app.config;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties("qqbot.security.app-secret")
public class AppSecretEncryptionProperties {

    private String keyId = "primary";
    private String masterKey = "";
    private String masterKeyFile = "";

    public String getKeyId() {
        return keyId;
    }

    public void setKeyId(String keyId) {
        this.keyId = keyId;
    }

    public String getMasterKey() {
        return masterKey;
    }

    public void setMasterKey(String masterKey) {
        this.masterKey = masterKey;
    }

    public String getMasterKeyFile() {
        return masterKeyFile;
    }

    public void setMasterKeyFile(String masterKeyFile) {
        this.masterKeyFile = masterKeyFile;
    }
}

File: qqbot-module-qqbot-runtime/src/main/kotlin/com/mieai/qqbot/app/config/AppSecretEncryptionProperties.kt
package com.mieai.qqbot.app.config

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties("qqbot.security.app-secret")
class AppSecretEncryptionProperties {
    var keyId: String = "primary"
    var masterKey: String = ""
    var masterKeyFile: String = ""
}
